remote-api/src/main/java/org/alfresco/opencmis/CMISServletDispatcher.java

```diff
@@ -133,9 +135,12 @@ public void setCmisVersion(String cmisVersion)
         this.cmisVersion = CmisVersion.fromValue(cmisVersion);
     }
 
-    public void setNonAttachContentTypes(Set<String> nonAttachWhiteList)
+    public void setNonAttachContentTypes(String nonAttachAllowListStr)
     {
-        this.nonAttachContentTypes = nonAttachWhiteList;
+		if (nonAttachAllowListStr != null) 
+		{
+			nonAttachContentTypes = Stream.of(nonAttachAllowListStr.trim().split("\\s*,\\s*")).collect(Collectors.toSet());

```

Comment:
Wouldn't following be a bit simpler?
```suggestion
			nonAttachContentTypes = Set.of(nonAttachAllowListStr.split(","));
```

Reply:
`.split("\\s*,\\s*")` Is to remove trailing and leading whitespace? Probably needs to stay, makes suggestion:
```suggestion
			nonAttachContentTypes = Set.of(nonAttachAllowListStr.split("\\s*,\\s*"));
```

Reply:
I did look at a few options. There are other parts of code base that use Google Commons (which I discounted).

As per @PunchedChimera it was related to ensuring we trim (potential) whitespace either side of comma as well as start and end of string. I realise that it is unlikely that this config will be overridden other than maybe option to have an empty list.

Reply:
I've updated based on your feedback (+ extra tweak). Thanks both 👍